components/page.js

```diff
@@ -40,6 +45,7 @@ export default ({
   children,
   navbarItems,
   isFullscreen,
+  isFullview,
```

Comment:
Seeing as `isFullscreen` was something that we created just for Hacker Eval, it's fine to replace it completely (assuming the reason you made `isFullview` was to retain behaviour for pages that use `isFullscreen`). No other pages in Admin use it, and it would prevent this page component from becoming more complicated.

Reply:
Oh fire I was wondering about this TY